### ISSUE TITLE
Existing keys should not be overwritten

### ISSUE THREAD

**Issue by iromli** [GluuFederation/docker-config-init#8]
Currently, `generate` and `load` commands will overwrite existing config. There should be ways to guard existing keys being overwritten accidentally.

1. Prompt user if key is exist
2. Add option to overwrite all keys

**Comment by iromli** [comment]
> 1. Prompt user if key is exist

Prompting user to override value of existing key may be dangerous as it could break related keys.

For example (assuming `encoded_salt` and `encoded_ox_ldap_pw` keys already exist in Consul):

```python
cfg["encoded_salt"] = get_random_chars(24)
cfg["encoded_ox_ldap_pw"] = encrypt_text(admin_pw, cfg["encoded_salt"])
```

 If somehow user choose to override `encoded_salt` but skipping `encoded_ox_ldap_pw`, then the `encoded_ox_ldap_pw` value cannot be decoded using new `encoded_salt` value.

In my opinion, we should not prompt user to change the value of existing key.

**Comment by iromli** [comment]
> 2. Add option to overwrite all keys

This feature is already in `config-init` as it's the default behavior.

**Comment by iromli** [comment]
What we may need is a feature to skip existing key being overwritten.

**Comment by iromli** [comment]
> We should generally skip overwriting unless explicitly given consent.

There are currently about 135 keys right now. Giving a flag to each of them will polute the script.

**Comment by iromli** [comment]
The latest `gluufederation/config-init:3.1.3_dev` image  implements these features during `generate` and `load` commands:

- key will be added if not exist
- existing key will be skipped
- overwrite all if env var `GLUU_OVERWRITE_ALL="true"` is passed

**Comment by iromli** [comment]
Merged to `master` branch and `config-init:3.1.3_04` and `config-init:latest` images.
